libairspy/src/airspy.c

```diff
@@ -27,6 +27,12 @@ ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSI
 #include <stdlib.h>
 #include <string.h>
 #include <libusb.h>
+
+#ifdef _WIN32
+#define HAVE_STRUCT_TIMESPEC 1
+#define strdup _strdup
+#endif
```

Comment:
Could you clarify this fix was intended only to fix issue with Visual Studio 2017/2019 or more  ?
- Such fix is not required for GNU based compiler on Windows like MinGW, MSYS2/mingw64 ...
- The actual source code do not use "strdup" so I do not understand why you add this #define strdup _strdup

I prefer a fix like that:
```
#if _MSC_VER > 1700  // To avoid error with Visual Studio 2017/2019 or more define which define timespec as it is already defined in pthread.h
#define HAVE_STRUCT_TIMESPEC
#endif
```

I'm waiting your feedback to have more details on your original issue (exact compiler ...)

Reply:
Yes you are totally right - I am compiling with Visual Studio 2019. I applied your suggested fix and it works as well thanks for the suggestion. I updated the PR accordingly.

Reply:
Thanks for your contribution